<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizerWidget</class>
 <widget class="QWidget" name="EqualizerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>320</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="margin">
    <number>12</number>
   </property>
   <item>
    <widget class="EqualizerCurveWidget" name="curveWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="bandLayout0">
       <item>
        <widget class="QLabel" name="labelFreq0">
         <property name="text">
          <string>31 Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand0">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue0">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout1">
       <item>
        <widget class="QLabel" name="labelFreq1">
         <property name="text">
          <string>62 Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand1">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue1">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout2">
       <item>
        <widget class="QLabel" name="labelFreq2">
         <property name="text">
          <string>125 Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand2">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue2">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout3">
       <item>
        <widget class="QLabel" name="labelFreq3">
         <property name="text">
          <string>250 Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand3">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue3">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout4">
       <item>
        <widget class="QLabel" name="labelFreq4">
         <property name="text">
          <string>500 Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand4">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue4">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout5">
       <item>
        <widget class="QLabel" name="labelFreq5">
         <property name="text">
          <string>1 kHz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand5">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue5">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout6">
       <item>
        <widget class="QLabel" name="labelFreq6">
         <property name="text">
          <string>2 kHz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand6">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue6">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout7">
       <item>
        <widget class="QLabel" name="labelFreq7">
         <property name="text">
          <string>4 kHz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand7">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue7">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout8">
       <item>
        <widget class="QLabel" name="labelFreq8">
         <property name="text">
          <string>8 kHz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand8">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue8">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bandLayout9">
       <item>
        <widget class="QLabel" name="labelFreq9">
         <property name="text">
          <string>16 kHz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBand9">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tickInterval">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue9">
         <property name="text">
          <string>0 dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EqualizerCurveWidget</class>
   <extends>QWidget</extends>
   <header>EqualizerCurveWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
